docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,56 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
```

Comment:
I wouldn't throw the docs away either-- just break them out into another PR that we can sit on for a while.